### PR TITLE
Updated OpenShift instances to {product-title} across Using Images

### PR DESCRIPTION
**Pull request by ahardin-rh**
Omitted xpass_images topics (addressing in a separate PR)

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tnguyen-rh @tpoitras PTAL

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -214,17 +214,24 @@ slave.
...
+the {product-title} server and the API artifacts hosted there.
+
+In addition to being accessible from the classic "freestyle" form of Jenkins
+job, the build steps as of version 1.0.14 of the {product-title} Pipeline
+Plug-in are also avaible to Jenkins Pipeline jobs via the DSL extension points
+provided by the Jenkins Pipeline Plug-in.
+
+The
+https://github.com/openshift/jenkins/tree/master/1/contrib/openshift/configuration/jobs/OpenShift%20Sample[sample
+Jenkins job] that is pre-configured in the Jenkins image utilizes the OpenShift
```
Comment (tpoitras) [inline using_images/other_images/jenkins.adoc]: I think ya missed one here "Jenkins image utilizes the OpenShift pipeline plug-in"?

**Comment by tpoitras** [comment]
@ahardin-rh 
i might have found one you missed, unless that was left unchanged intentionally.
Other than that, LGTM! :+1: :ok:

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (ahardin-rh) [inline using_images/other_images/jenkins.adoc]: Good catch!  :star:  Thanks!

**Comment by ahardin-rh** [comment]
No rev history needed

**Pull request merged**